src/test/steps/signUpSteps.ts

```diff
@@ -0,0 +1,17 @@
+import { When, Then } from '@cucumber/cucumber';
+import { expect } from '@playwright/test';
+import basePage from '../../pages/basePage';
+
+When('User navigates to sign up page', async function () {
+  await basePage.signUpPage.navigateToRegisterPage();
+});
```

Comment:
What if the user wants to navigate to another page? Are you going to create another step definition for that?